src/cascadia/TerminalApp/KeyChordSerialization.cpp

```diff
@@ -240,12 +250,18 @@ winrt::hstring KeyChordSerialization::ToString(const KeyChord& chord)
     }
     else
     {
-        if (vkeyNamePairs.find(vkey) != vkeyNamePairs.end())
+        bool foundKey = false;
+        for (const auto& pair : vkeyNamePairs)
         {
-            buffer += vkeyNamePairs.at(vkey);
-            serializedSuccessfully = true;
+            if (pair.second == vkey)
+            {
+                buffer += pair.first;
+                serializedSuccessfully = true;
+                foundKey = true;
+                break;

```

Comment:
i'm glad you made the _serialization_ more expensive honestly - we shouldn't be using it often. parsing happens all the dang time.